builtin/docker/builder.go

```diff
@@ -322,6 +328,7 @@ func (b *Builder) buildWithDocker(
 	tag string,
 	platform string,
 	buildArgs map[string]*string,
+	Target string,
```

Comment:
This should be `target`

```suggestion
	target string,
```